docs/src/showcase/bayesian_neural_ode.md

```diff
@@ -133,7 +132,8 @@ pl = scatter(tsteps, ode_data[1, :], color = :red, label = "Data: Var1", xlabel
              title = "Spiral Neural ODE")
 scatter!(tsteps, ode_data[2, :], color = :blue, label = "Data: Var2")
 for k in 1:300
-    resol = predict_neuralode(samples[:, 100:end][:, rand(1:400)])
+    newp = typeof(p)(samples[:, 100:end][:, rand(1:400)])
+    resol = predict_neuralode(newp)
```

Comment:
@avik-pal @Vaibhavdixit02 do you know off the top of your head what kind of object this is returning? I don't have a second to actually run this.